jaxlib/BUILD

```diff
@@ -76,13 +76,14 @@ pyx_library(
 py_library(
     name = "jaxlib",
     srcs = [
-        "cuda_prng.py",
-        "cusolver.py",
         "init.py",
         "pocketfft.py",
         "setup.py",
         "version.py",
-    ],
+    ] + if_cuda([
```

Comment:
This change actually causes problems for our internal CI. I think it might work with `if_cuda_is_configured`, although you'll need to verify that keeps the ROCm version working.

But also see my comments on the next change: ideally I think we can just share these python files?

Reply:
As long as 'gpu' does not default to 'ROCM' in xla_client.py we can omit this whole commit

Reply:
@hawkinsp Lets just leave in cusolver.py and live with the `cusolver_kernels is not defined` errors until we fix it in https://github.com/google/jax/pull/5115

Reply:
Or if you deem it necessary we could also write a dummy version which returns NotImplementedError

Reply:
Resolving this in the next PR seems fine. This is all new hardware support so it doesn't break anything!